src/main/java/racinggame/view/InputView.java

```diff
@@ -1,66 +1,61 @@
 package racinggame.view;
 
-import racinggame.domain.Utils;
+import static racinggame.exception.ExceptionBaseMessage.EXCEEDED_RETRIES_EXCEPTION_MESSAGE;
+
+import java.util.Scanner;
 import racinggame.domain.Validator;
+import racinggame.exception.InputBlankException;
 
 public class InputView {
 
+    public static final String QUIT = "q";
     private static final String SEPARATOR = ",";
     private static final String BLANK = "\\s+";
     private static final String EMPTY_STRING = "";
 
-    private InputView() {}
-
-    public static String[] getCarNames() {
-        String[] carNames;
-
-        while (true) {
-            String inputCarNames = inputCarNames();
-            carNames = splitNameByComma(inputCarNames);
+    private static int tryCount = 1;
 
-            try {
-                Validator.validatePossibleToStart(carNames);
-                break;
-            } catch (IllegalArgumentException e) {
-                System.out.println(e.getMessage());
-            }
-        }
+    private InputView() {}
 
-        return carNames;
+    public static String inputCarNames() {
+        System.out.printf("경주할 자동차 이름을 %s로 구분해서 입력하세요.(%d대 이상)\n", SEPARATOR, Validator.MINIMUM_PLAYER);
+        return getInput();
     }
 
-    private static String inputCarNames() {
-        System.out.println(String.format("경주할 자동차 이름을 '%s'로 구분해서 입력하세요.(%d대 이상)", SEPARATOR, Validator.MINIMUM_PLAYER));
-        return removeBlank(Utils.getInput());
-    }
+    public static String inputTrial() throws IllegalAccessException {
+        System.out.print("시도 횟수를 입력해 주세요: ");
+        String input = removeBlank(getInput());
+
+        try {
+            Validator.checkIsBlank(input);
+            Validator.isNumber(input);
+        } catch (InputBlankException | NumberFormatException e) {
+            System.out.println(e.getMessage());
+            input = retryInputTrial();
+        }
```

Comment:
```suggestion
        Validator.checkIsBlank(input);
        Validator.isNumber(input);
```
재시작로직에게 메시지를 던지는 방식으로 수정하는 것은 어떨까요?

Reply:
반영하였습니다!! 바깥의 retry 로직과는 별개로 작동하기 위해 해당 부분에서 던져주는 예외는 밖의 retry 로작에서 잡지 않도록 처리하였습니다.